db/migrate/20190613134100_convert_to_active_storage.rb

```diff
@@ -25,6 +25,13 @@ def up
 
     transaction do
       models.each do |model|
+        table_exists = ActiveRecord::Base.connection.execute("SELECT EXISTS (
+          SELECT 1
+          FROM   information_schema.tables
+          WHERE  table_name = '#{model.table_name}'
+        )").as_json[0]['exists']
+        next unless table_exists
+
```

Comment:
next unless ActiveRecord::Base.connection.table_exists?(model.table_name)

Reply:
Thank you, fixed.